docs/recipes.md

```diff
@@ -0,0 +1,309 @@
+# Recipes
+
+This page includes code snippets or "**recipes**" for a variety of common tasks.
+Use them as building blocks or examples when making your own notebooks.
+
+In these recipes, **each code block represents a cell**.
+ 
+## Control Flow
+
+### Show an output conditionally
+
+**Task.** Only show an output if a condition is `True`.
```

Comment:
I think recipes should be more "use-case" oriented and include "why" you could do this:

"For example, you may want to wait for user input or lazily load data"